frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -75,10 +93,17 @@ export class TemplateValidations {
 
   getDefaultBus = (defaultBus = DiskBus.VIRTIO): DiskBus => {
```

Comment:
IMO, this would be more readable

```tsx

  getDefaultBus = (defaultBus = DiskBus.VIRTIO): DiskBus => {
    const allowedBuses = this.getAllowedBuses();

    if (allowedBuses.size === 0) {
      return defaultBus;
    }

    const recommendedBuses = this.getRecommendedBuses();

    if (recommendedBuses.has(defaultBus)) {
      return defaultBus;
    }

    if (recommendedBuses.size > 0) {
      return [...recommendedBuses][0];
    }

    return allowedBuses.has(defaultBus) ? defaultBus : [...allowedBuses][0];
  };

```